server/src/main/java/org/hyperskill/community/flashcards/config/MongoConfiguration.java

```diff
@@ -1,16 +1,24 @@
 package org.hyperskill.community.flashcards.config;
 
 import com.mongodb.client.MongoClients;
+import org.springframework.beans.factory.annotation.Qualifier;
 import org.springframework.context.annotation.Bean;
 import org.springframework.context.annotation.Configuration;
+import org.springframework.data.mongodb.config.EnableMongoAuditing;
 import org.springframework.data.mongodb.core.MongoTemplate;
 
 @Configuration
+@EnableMongoAuditing
 public class MongoConfiguration {
 
     @Bean
     public MongoTemplate mongoTemplate() {
         return new MongoTemplate(MongoClients.create(), "cards");
     }
 
+    @Bean
+    @Qualifier("example")
+    public MongoTemplate exampleMongoTemplate() {
+        return new MongoTemplate(MongoClients.create(), "example");
+    }
 }
```

Comment:
This bean creates the example database.
It's not was I originally intendend - as I thought more of creating a collection named "example" within the cards database.

But we can discuss this anyway

Reply:
Let's keep it for now, I'll remove it when the card and category models are finalized.